Formula/fceux.rb

```diff
@@ -1,11 +1,14 @@
+# typed: false
+# frozen_string_literal: true
+
+# Formula for fceux
```

Comment:
Please remove this. When you do so, squash this change with your existing commit and amend the commit message to read
```
fceux 2.4.0
```
Thanks!

Reply:
Did I do the squash correctly?

Reply:
would it be easier if I just start over and make a new pull request? I don't know why I can't fix the history with git rebase.

Reply:
No need; cleaned that up for you. Whenever you `git rebase` locally to change history on a remote, you need `git push --force`. (The `--force` flag is to remind you to avoid doing it on branches other people are using.)